test/tests.py

```diff
@@ -187,36 +224,38 @@ def test_0004_negativeAuth_invalidCreds(self):
 
             credentials = [
                 (username, password),
-                (self.username, password),
-                (username, self.password)
+                (self.source_username, password),
+                (username, self.source_password)
             ]
 
             for user, pw in credentials:
-                response = requests.get(self.proxy_endpoint, auth=(user, pw), verify=False)
+                response = requests.get(self.proxy_endpoint, auth=(user, pw), verify=self.source_verify_ssl)
                 self.assertEqual(response.status_code, HTTPStatus.UNAUTHORIZED)
 
-    def test_0005_negativeAuth_missingCreds(self):
+    def test_0004_negativeAuth_missingCreds(self):
         # This test will use no credentials at all
         # With an empty authorization header
-        response = requests.get(self.proxy_endpoint, auth=('', ''), verify=False)
+        response = requests.get(self.proxy_endpoint, auth=('', ''), verify=self.source_verify_ssl)
         self.assertEqual(response.status_code, HTTPStatus.UNAUTHORIZED)
 
         # Without an authorization header.
-        response = requests.get(self.proxy_endpoint, verify=False)
+        response = requests.get(self.proxy_endpoint, verify=self.source_verify_ssl)
         self.assertEqual(response.status_code, HTTPStatus.UNAUTHORIZED)
 
-    def test_0006_invalidIncorrectUri(self):
+    def test_0005_invalidIncorrectUri(self):
         # This test will send an invalid URI
         invalidUri = "/invalidURI"
-        response = requests.get(f'{self.proxy_endpoint}{invalidUri}', auth=self.auth, verify=False)
+        response = requests.get(f'{self.proxy_endpoint}{invalidUri}', auth=self.source_auth,
+                                verify=self.source_verify_ssl)
         self.assertEqual(response.status_code, HTTPStatus.NOT_FOUND)
 
         # This test will send an incorrect URI
         incorrectUri = "/_cluster/incorrectUri"
-        response = requests.get(f'{self.proxy_endpoint}{incorrectUri}', auth=self.auth, verify=False)
+        response = requests.get(f'{self.proxy_endpoint}{incorrectUri}', auth=self.source_auth,
+                                verify=self.source_verify_ssl)
         self.assertEqual(response.status_code, HTTPStatus.METHOD_NOT_ALLOWED)
 
-    def test_0007_OSB(self):
+    def test_0006_OSB(self):
```

Comment:
This seems like it will need to be tweaked to work for AWS as well.  For AWS we will already be on the migration console box so we would just need to make the `runTestBenchmarks.sh` call from the right directory

Reply:
You're right, this specific test won't work on AWS as of this moment. I imagined that this script will end up existing on the migration console and running from there. I can update this test in a future PR to support that.
As for the other tests; per my testing on our deployed AWS solution (w/o SigV4 whatsoever), they will pass.